src/client/client_http.go

```diff
@@ -108,14 +108,19 @@ func DownloadNetwork(httpClient *http.Client, hostname string, networkPath strin
 	if err != nil {
 		return err
 	}
-	defer r.Body.Close()
 
-	out, err := os.Create(networkPath)
+	out, err := ioutil.TempFile("networks", sha + "_tmp")
```

Comment:
You had a commit with comment that "networks" is no longer hard coded, but its still hard coded here.

Reply:
Borgs comment that you could use path util to get the directory out of networkPath seems a reasonable choice.

Reply:
This is the one that I was mostly worried about. My (untested) suggestion was:
```
directory, _ := filepath.split(networkPath)
out, err := ioutil.TempFile(directory, sha + "_tmp")
```
I would prefer something like that over changing the function signature since it is used by the old client as well.

Reply:
oh right I missed that. fixed